packages/web-components/src/components/dropdown/dropdown.mdx

```diff
@@ -1,89 +1,183 @@
+action to filter or sort existing content.
 
-### JS (via import)
+<Canvas of={DropdownStories.Default} />
 
-```javascript
-import '@carbon/web-components/es/components/dropdown/index.js';
-```
+### Inline Dropdown
 
-<Markdown>{`${cdnJs({ components: ['dropdown'] })}`}</Markdown>
-<Markdown>{`${cdnCss()}`}</Markdown>
+You can place a `cds-dropdown` inline with other content by using the `inline`
+variant
+
+<Canvas of={DropdownStories.Inline} />
+
+### Skeleton state
+
+You can use the `cds-dropdown-skeleton` component to render a skeleton variant
+of a dropdown. This is useful to display on initial page load or when data needs
+to be fetched from an external API.
 
-### HTML
+<Canvas of={DropdownStories.Skeleton} />
+
+### AI label
+
+<Canvas of={DropdownStories.WithAILabel} />
+
+## Component API
+
+## `cds-dropdown`
+
+<ArgTypes of="cds-dropdown" />
+
+## `cds-dropdown-item`
+
+<ArgTypes of="cds-dropdown-item" />
+
+### Dropdown `direction`
+
+When a dropdown is near the bottom of a page, you may want to render the menu to
+the top of the screen. To do this, specify `direction="top"`
+
+<cds-dropdown direction="top" label="Select an option...">
+  <cds-dropdown-item value="0">Option 1</cds-dropdown-item>
+  <cds-dropdown-item value="1">Option 2</cds-dropdown-item>
+  <cds-dropdown-item value="2">Option 3</cds-dropdown-item>
+</cds-dropdown>
 
 ```html
-<cds-dropdown value="bar">
-  <cds-dropdown-item value="foo">Foo</cds-dropdown-item>
-  <cds-dropdown-item value="bar">Bar</cds-dropdown-item>
-  <cds-dropdown-item value="baz">Baz</cds-dropdown-item>
+<cds-dropdown direction="top" label="Select an option...">
+  <cds-dropdown-item value="0">Option 1</cds-dropdown-item>
+  <cds-dropdown-item value="1">Option 2</cds-dropdown-item>
+  <cds-dropdown-item value="2">Option 3</cds-dropdown-item>
 </cds-dropdown>
 ```
 
-## Selection
+### Dropdown `value`
 
-When user attempts to select a dropdown item, `cds-dropdown-beingselected` event
-fires on `<cds-dropdown>`, which has the following properties in the event
-details:
+If you want the dropdown to render with an option already selected, but do not
+want to fully control the component, you can set the dropdown `value` to the
+option you want selected.
 
-| Property | Description                               |
-| -------- | ----------------------------------------- |
-| `item`   | The `<cds-dropdown-item>` being selected. |
+<cds-dropdown value="option-0">
+  <cds-dropdown-item value="option-0">
+    Initially selected option
+  </cds-dropdown-item>
+  <cds-dropdown-item value="option-1">Option 1</cds-dropdown-item>
+  <cds-dropdown-item value="option-2">Option 2</cds-dropdown-item>
+</cds-dropdown>
 
-`cds-dropdown-beingselected` bubbles and is cancelable. Canceling this event
-means canceling change in selection.
+```html
+<cds-dropdown value="option-0">
+  <cds-dropdown-item value="option-0">
+    Initially selected option
+  </cds-dropdown-item>
```

Comment:
something about this not being one line is causing the dropdown to appear like: 
<img width="1090" alt="Screenshot 2025-04-18 at 9 54 40 AM" src="https://github.com/user-attachments/assets/35858886-fdd1-487f-9f54-6c52cb893bad" />

Reply:
Good catch! Seems like `prettier` force-formatting this line is adding a `<p>` due to the line breaks. Shortening the value to `Initially selected` puts it all on one line and fixes it.

```html
<cds-dropdown-item value="option-0">
  Initially selected option
</cds-dropdown-item>
```